src/redux/thunk.js

```diff
@@ -56,28 +56,26 @@ const createUserThunk = user => {
     dispatch(createUsers(response));
   };
 };
-const updateUserThunk = (user)=> {
-  return async (dispatch)=> {
+const updateUserThunk = user => {
+  return async dispatch => {
     try {
       const updatedUser = (await axios.put(`/api/users/${user.id}`, user)).data;
       dispatch(updateUser(updatedUser));
```

Comment:
please add 'updateUser' to imports on line 10. You had it on the other PR but forgot it here.